README.md

```diff
@@ -27,6 +27,13 @@ detailed instructions, see Waku v1 and v2 home above.
 
 More information on the installation of these can be found [here](https://github.com/status-im/nimbus#prerequisites).
 
+### Clone
+
+```bash
+# Ensure submodules are cloned as well
+git clone --recurse-submodules <REPOSITORY_REMOTE_URL>
```

Comment:
Thanks, indeed missing. Can we add it right before `make wakunode1 wakunode2` in "Wakunode" section?

`make update` should e fine instead.

Reply:
yup as @LNSD I think we should just add `make update`

Reply:
@felicio Mind adding `make update`, so we can merge this? Thanks!